tests/StatusBar.test.ts

```diff
@@ -42,26 +67,12 @@ describe('StatusBar', () => {
       const helpers = statusBar.bind(source)
       ;['initial', 'running', 'success', 'failed', 'stopped'].forEach(status => {
         helpers[status]()
-        expect((statusBar as any).queue).toContainEqual({ source, status })
+        expect(updateStatusSpy).toHaveBeenCalledWith({ source, status })
```

Comment:
Are you sure that this is working? `{ source, status }` creates a new object and I thought that `toHaveBeenCalledWith` only checks identity of the reference (in contrast to `toContainEqual`)

Reply:
as far as jest 24.7.0 goes, yes it worked, otherwise, the test will fail 😏